packages/editor/src/components/inserter/menu.js

```diff
@@ -48,14 +49,29 @@ const MAX_SUGGESTED_ITEMS = 9;
  * @return {Array}             Filtered item list.
  */
 export const searchItems = ( items, searchTerm ) => {
-	const normalizedSearchTerm = searchTerm.toLowerCase().trim();
-	const matchSearch = ( string ) => string.toLowerCase().indexOf( normalizedSearchTerm ) !== -1;
+	const normalizedSearchTerm = normalizeTerm( searchTerm );
+	const matchSearch = ( string ) => normalizeTerm( string ).indexOf( normalizedSearchTerm ) !== -1;
 
 	return items.filter( ( item ) =>
 		matchSearch( item.title ) || some( item.keywords, matchSearch )
 	);
 };
 
+/**
+ * Converts the search term into a normalized term, removing diacritics
+ * and whitespace.
```

Comment:
To be clearer, is it only removing leading and trailing whitespace, or all whitespace?

Reply:
Thanks, @aduth I've updated the comment message to add clarity.